src/frequenz/dispatch/_dispatcher.py

```diff
@@ -183,6 +184,8 @@ def __init__(
             server_url: The URL of the dispatch service.
             key: The key to access the service.
         """
+        super().__init__(name="Dispatcher")
```

Comment:
It is always better to either forward the argument or leave it empty. Creating 2 dispatcher will result in 2 instances having the same name/unique ID, which will be super confusing.